plugin/cmd/kubectl/binding/plugin.yaml

```diff
@@ -0,0 +1,10 @@
+name: "binding"
+shortDesc: "This command interacts with Service Bindings"
+command: "./binding"
```

Comment:
leave off the "./" since you don't now how they'll specify the path to the exe (if I'm understanding how this is being used)

Reply:
Nah, that needs to be there for kubectl to know how to invoke the executable - it will need to be different for these to work on windows, but that has other issues.